source/develop/release-management/features/network/lldp.html.md

```diff
@@ -0,0 +1,448 @@
+```
+
+The response in XML for the [SampleCaptures from Wireshark's wiki][5] will look like this
+
+```xml
+<host_nic_neighbor>
+  <host_nic href="/ovirt-Engine/api/hosts/bdb7c9f9-e10d-4af0-af9c-a7f0c7235fdd/nics/0ac967ac-af2c-4628-997d-ef845edeeeb0" id="0ac967ac-af2c-4628-997d-ef845edeeeb0"/>
+  <tlvs>
+    <tlv>
+      <name>Chassis ID</name>
+      <properties>
+        <property>
+          <name>chassis ID subtype</name>
+          <value>MAC address</value>
+        </property>
+        <property>
+          <name>chassis ID</name>
+          <value>00:01:30:f9:ad:a0</value>
+        </property>
+      </properties>
+      <tlv_type>0x1</tlv_type>
+    </tlv>
+    <tlv>
+      <name>Port ID</name>
+      <properties>
+        <property>
+          <name>port ID subtype</name>
+          <value>Interface Name</value>
+        </property>
+        <property>
+          <name>port ID</name>
+          <value>1/1</value>
+        </property>
+      </properties>
+      <tlv_type>0x2</tlv_type>
+    </tlv>
+    <tlv>
+      <name>Time to Live</name>
+      <properties>
+        <property>
+          <name>TTL</name>
+          <value>120</value>
+        </property>
+      </properties>
+      <tlv_type>0x3</tlv_type>
+    </tlv>
+    <tlv>
+      <name>Port Description</name>
+      <properties>
+        <property>
+          <name>port description</name>
+          <value>Summit300-48-Port 1001</value>
+        </property>
+      </properties>
+      <tlv_type>0x4</tlv_type>
+    </tlv>
+    <tlv>
+      <name>System Name</name>
+      <properties>
+        <property>
+          <name>system name</name>
+          <value>Summit300-48</value>
+        </property>
+      </properties>
+      <tlv_type>0x5</tlv_type>
+    </tlv>
+    <tlv>
+      <name>System Description</name>
+      <properties>
+        <property>
+          <name>system description</name>
+          <value>Summit300-48 - Version 7.4e.1 (Build 5) by Release_Master 05/27/05 04:53:11</value>
+        </property>
+      </properties>
+      <tlv_type>0x6</tlv_type>
+    </tlv>
+    <tlv>
+      <name>System Capabilities</name>
+      <properties>
+        <property>
+          <name>system capabilities</name>
+          <value>Bridge, Router</value>
+        </property>
+        <property>
+          <name>enabled capabilities</name>
+          <value>Bridge, Router</value>
+        </property>
+      </properties>
+      <tlv_type>0x7</tlv_type>
+    </tlv>
+    <tlv>
+      <name>Management Address</name>
+      <properties>
+        <property>
+          <name>management address subtype</name>
+          <value>AFI 802</value>
+        </property>
+        <property>
+          <name>management address</name>
+          <value>00:01:30:f9:ad:a0</value>
+        </property>
+      </properties>
+      <tlv_type>0x8</tlv_type>
+    </tlv>
+    <tlv>
+      <name>Port VLAN Id</name>
+      <oui>0x0080c2</oui>
```

Comment:
Not a must, but it would be nice if it will be possible to add an attribute that tells us the name of this id.
<oui name="bla">0x00..</oui>